snuba/request/factory.py

```diff
@@ -0,0 +1,26 @@
+import uuid
+from typing import Any
+
+from snuba.attribution.attribution_info import AttributionInfo
+from snuba.query.composite import CompositeQuery
+from snuba.query.data_source.simple import Entity
+from snuba.query.logical import Query
+from snuba.query.query_settings import QuerySettings
+from snuba.request import Request
+
+
+def create_request(
```

Comment:
did you need all this code just to not have to pass in a uuid?

Reply:
fair point, I changed it to just pass in the uuid